java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -183,4 +185,103 @@ CompletableFuture<Double> zaddIncr(
      * }</pre>
      */
     CompletableFuture<Long> zcard(String key);
+
+    /**
+     * Returns the specified range of elements in the sorted set stored at <code>key</code>.<br>
+     * ZRANGE can perform different types of range queries: by index (rank), by the score, or by
+     * lexicographical order.<br>
+     * To get the elements with their scores, see zrange_withscores.
+     *
+     * @see <a href="https://redis.io/commands/zrange/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param rangeQuery The range query object representing the type of range query to perform.<br>
+     *     - For range queries by index (rank), use RangeByIndex.<br>
+     *     - For range queries by lexicographical order, use RangeByLex.<br>
+     *     - For range queries by score, use RangeByScore.
+     * @param reverse If true, reverses the sorted set, with index 0 as the element with the highest
+     *     score.
+     * @return An array elements within the specified range. If <code>key</code> does not exist, it is
+     *     treated as an empty sorted set, and the command returns an empty array.
+     * @example
+     *     <pre>{@code
+     * String[] payload1 = client.zrange("mySortedSet", new RangeByIndex(0, -1), true).get();
```

Comment:
The -1 isn't described in our documentation.  But it is described on the Redis side.  I'm worried about including an example that isn't described, but also I don't want to include too many details in our own documentation/.